src/CFNetwork/CFHost.cs

```diff
@@ -28,26 +28,26 @@ namespace CFNetwork {
 namespace CoreServices {
 #endif
 
-#if !NET
-	[Deprecated (PlatformName.WatchOS, 8, 0, message: Constants.UseNetworkInstead)]
-	[Deprecated (PlatformName.TvOS, 15, 0, message: Constants.UseNetworkInstead)]
- 	[Deprecated (PlatformName.iOS, 15, 0, message: Constants.UseNetworkInstead)]
- 	[Deprecated (PlatformName.MacCatalyst, 15, 0, message: Constants.UseNetworkInstead)]
- 	[Deprecated (PlatformName.MacOSX, 12, 0, message: Constants.UseNetworkInstead)]
-#else
-	[UnsupportedOSPlatform ("ios15.0")]
+#if NET
 	[UnsupportedOSPlatform ("tvos15.0")]
 	[UnsupportedOSPlatform ("maccatalyst15.0")]
 	[UnsupportedOSPlatform ("macos12.0")]
-#if __MACCATALYST__
-	[Obsolete ("Starting with maccatalyst15.0 use 'Network.framework' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
-#elif IOS
-	[Obsolete ("Starting with ios15.0 use 'Network.framework' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
-#elif TVOS
-	[Obsolete ("Starting with tvos15.0 use 'Network.framework' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+	[UnsupportedOSPlatform ("ios15.0")]
+#if TVOS
+	[Obsolete ("Starting with tvos15.0 onstants.UseNetworkInstea.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#elif __MACCATALYST__
+	[Obsolete ("Starting with maccatalyst15.0 onstants.UseNetworkInstea.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
 #elif MONOMAC
-	[Obsolete ("Starting with macos12.0 use 'Network.framework' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+	[Obsolete ("Starting with macos12.0 onstants.UseNetworkInstea.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
```

Comment:
This consant isn't quite right...

Reply:
Let me look into what the heck mellite is smoking...

Reply:
Oh, it's a variable, and mellite just sees "Constants.UseNetworkInstead". Talk about a case I haven't seen before.

Reply:
I have corrected this by teaching the tool to look for constants and spitting out a build breaking [Verify], which needs manual fixing.

Reply:
https://github.com/chamons/mellite/commit/226eb1f9ede3e6f152a194b3c1fef52910f620e6